Runtime/CSharp/Bitcraft.StateMachine/EventArgs.cs

```diff
@@ -25,21 +30,28 @@ public class StateEnterEventArgs : EventArgs
         /// <summary>
         /// Initializes the StateEnterEventArgs instance.
         /// </summary>
+        /// <param name="triggeringAction">Action token of the action that triggered the transition.</param>
```

Comment:
For consistency, "The action token of the action that triggered the transition.".
My bad.